src/directLine.ts

```diff
@@ -252,6 +261,7 @@ export interface IBotConnection {
     connectionStatus$: BehaviorSubject<ConnectionStatus>,
     activity$: Observable<Activity>,
     end(): void,
+    getConversation(): Conversation,
```

Comment:
My first thought was to expose the private fields, but then I realized that they probably shouldn't be writable.